src-django/authentication/tests/utils/factories.py

```diff
@@ -0,0 +1,23 @@
+from api.tests.utils import factories
+from authentication.models import EMAIL_CONFIRMATION_KEY_TTL_DAYS, UserProfile, EmailConfirmationKey
+import datetime
+import factory
+
+
+class UserProfileFactory(factory.django.DjangoModelFactory):
+    class Meta:
+        model = UserProfile
+        django_get_or_create = ('user',)
+
+    user = factory.SubFactory(factories.UserFactory)
+    is_email_confirmed = False
+
+
+class EmailConfirmationKeyFactory(factory.django.DjangoModelFactory):
+    class Meta:
+        model = EmailConfirmationKey
+        django_get_or_create = ('user',)
+
+    user = factory.SubFactory(factories.UserFactory)
+    key = '84bff1c64351a616828596f01b4e0d3155190eb5'
```

Comment:
How is this key being used? If it's to be used in a HMAC function, maybe we should use the secrete Django key?